andela/tesla-ah
# #166240990 Send a verification link after a successful registration

#### What does this PR do?
Send a verification link via email after a successful registration

#### Description of Task to be completed?
- User signs up
- A server validates the information
- If yes, it registers a user
- Send a verification link to the new user's email
- If a user clicks a link from an email, a server checks the user
- If the user is valid, VERIFICATION IS DONE

#### How should this be manually tested?
Clone the repository, this branch
- ```yarn install```
- Run ```sequelize db:migrate``` to run the migrations
- Run ```yarn dev``` to run the server
- Hit the endpoint ```/api/auth/signup``` , fill out the data and send the request. Use ```POST``` request.
- It will send an email, click ```click here to verify your account```
- It will take you to your default browser and you can get a response whether your account is verified or not.




#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166240990](https://www.pivotaltracker.com/story/show/166240990)

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2019-06-12 at 15 40 42" src="https://user-images.githubusercontent.com/22316636/59356190-a94b3f80-8d28-11e9-8a22-3f6c0d42f84b.png">

#### Questions:
N/A
